### PR TITLE
uuidgen: add page

### PR DESCRIPTION
**Pull request by asjadjawed**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by bl-ue** [comment]
@asjadjawed we actually already have a page for `uuidgen`: [`pages/linux/uuidgen.md`](https://github.com/tldr-pages/tldr/blob/master/pages/linux/uuidgen.md).  If the `uuidgen` you're writing a page for is the same as the one we already have a page for, then we should move that page to common instead of adding a new `osx` page.

**Comment by asjadjawed** [comment]
> @asjadjawed we actually already have a page for `uuidgen`: [`pages/linux/uuidgen.md`](https://github.com/tldr-pages/tldr/blob/master/pages/linux/uuidgen.md). If the `uuidgen` you're writing a page for is the same as the one we already have a page for, then we should move that page to common instead of adding a new `osx` page.

It is not exactly the same since `osx` has different arguments.

**Comment by bl-ue** [comment]
Ah, okay, good to know.  Thank you.

File: pages/osx/uuidgen.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# uuidgen
+
+> Generates new Universally Unique IDentifier UUID strings.
```
Comment (sbrl) [inline pages/osx/uuidgen.md:3]: Perhaps this will read better?
```suggestion
> Generate new UUID (Universally Unique IDentifier) strings.
```

**Comment by bl-ue** [comment]
@asjadjawed will you please add a more information link?

**Comment by asjadjawed** [comment]
more information link?

File: pages/osx/uuidgen.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# uuidgen
+
+> Generate new UUID (Universally Unique IDentifier) strings.
```
Comment (marchersimon) [inline pages/osx/uuidgen.md:3]:
```suggestion
> Generate new UUID (Universally Unique IDentifier) strings.
> More information: <https://www.ss64.com/osx/uuidgen.html>.
```

**Review by marchersimon — COMMENTED** [review]
A link to either the home page or a man page of the command. We have that in all of our pages.

**Review by bl-ue — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @asjadjawed!

**Pull request merged**